src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -1575,6 +1692,7 @@ private CloudApplication findApplication(UUID guid) {
                                                                                                .build());
     }
 
+    @Deprecated
```

Comment:
Should we add a TODO item?

Reply:
Yes 10x, added

Reply:
This would be great but there is an issue in the `cf-java-client` about the missing `include` parameter. So, for now, the only way to use the include parameter is via a curl request.